.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "useTabs": false,
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "all"
+}
```

Comment:
```
{
  "bracketSpacing": false,
  "jsxBracketSameLine": true,
  "printWidth": 80,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false
}
```

Reply:
Let's also add the rest of the config options. In particular I don't want `bracketSpacing` :)